src/sentry/integrations/vsts/integration.py

```diff
@@ -42,10 +44,12 @@ def get_project_config(self):
         disabled = False
         try:
             projects = client.get_projects(self.model.metadata['domain_name'])
-        except ApiError:
+        except Exception as e:
             # TODO(LB): Disable for now. Need to decide what to do with this in the future
             # should a message be shown to the user?
-            # Should we try refreshing the token? For VSTS that often clears up the problem
+            if INVALID_ACCESS_TOKEN == e.message:
```

Comment:
This check should happen in the API client as well right

Reply:
I'm not certain I would want it there. for instance, `get_project` is just:
```python
def get_projects(self, instance):
        # TODO(dcramer): VSTS doesn't provide a way to search, so we're
        # making the assumption that a user has 100 or less projects today.
        return self.get(
            VstsApiPath.projects.format(
                account_name=instance,
            ),
            params={'stateFilter': 'WellFormed'}
        )
```
This feels context specific to me. But since it can't do anything yet, maybe it's best to just remove it and have a to-do instead.